Formula/s/squiid.rb

```diff
@@ -0,0 +1,48 @@
+class Squiid < Formula
+  desc "Modular calculator written in Rust"
+  homepage "https://imaginaryinfinity.net/"
+  url "https://gitlab.com/ImaginaryInfinity/squiid-calculator/squiid/-/archive/1.0.6/squiid-1.0.6.tar.gz"
+  sha256 "785dd38de948c9e1fa8ae33a42d092bd21ab57e1399106c37d75962c1bf2ea92"
+  license "GPL-3.0-only"
+
+  depends_on "cmake" => :build
```

Comment:
[The dependency on the `nng-sys` crate](https://gitlab.com/ImaginaryInfinity/squiid-calculator/squiid/-/blob/1.0.6/squiid-engine/Cargo.toml?ref_type=tags#L13) is likely the reason why CMake is needed. However, vendored dependencies are generally discouraged, and we do have an `nng` formula.

I'll soon push a change to force linking with brewed `nng`.